bot/exts/evergreen/trivia_quiz.py

```diff
@@ -261,36 +523,48 @@ async def declare_winner(channel: discord.TextChannel, player_data: dict) -> Non
 
                 winners_mention = " ".join(winner.mention for winner in winners)
             else:
-                word = "You"
                 author_index = list(player_data.values()).index(highest_points)
                 winner = list(player_data.keys())[author_index]
                 winners_mention = winner.mention
 
             await channel.send(
                 f"Congratulations {winners_mention} :tada: "
-                f"{word} have won this quiz game with a grand total of {highest_points} points!"
+                f"You have won this quiz game with a grand total of {highest_points} points!"
             )
 
     def category_embed(self) -> discord.Embed:
         """Build an embed showing all available trivia categories."""
-        embed = discord.Embed(colour=discord.Colour.blue())
-        embed.title = "The available question categories are:"
-        embed.set_footer(text="If a category is not chosen, a random one will be selected.")
-        embed.description = ""
+        embed = discord.Embed(
+            colour=discord.Colour.blue(),
+            title="The available question categories are:",
+            description="",
+        )
+
+        embed.set_footer(
+            text="If a category is not chosen, a random one will be selected."
+        )
 
         for cat, description in self.categories.items():
-            embed.description += f"**- {cat.capitalize()}**\n{description.capitalize()}\n"
+            embed.description += (
```

Comment:
blacks formatting again not being ideal

Reply:
this one is from me too, for the same reason as the previous f-string literal concatenation. i thought it would improve readability, and this one is quite long too

Reply:
as discussed [here](https://discord.com/channels/267624335836053506/635950537262759947/841065351822311424), not reverting it. everything resolved :D